addons/web_editor/controllers/main.py

```diff
@@ -124,9 +124,7 @@ def export_icon_to_png(self, icon, color='#000', bg=None, size=100, alpha=255, f
         image = Image.new("RGBA", (width, height), color)
         draw = ImageDraw.Draw(image)
 
-        box = draw.textbbox((0, 0), icon, font=font_obj)
-        boxw = box[2] - box[0]
-        boxh = box[3] - box[1]
+        boxw, boxh = draw.textsize(icon, font=font_obj)
```

Comment:
Yes you are right we didn't notice that because of the requirements but it is not correct.

Reverting it is not correct either since it won't work on latest python versions/ubuntu noble but we should make it conditional, based on pillow version or using hasattr maybe. 

```suggestion
        if hasattr(draw, 'textbbox'):
            box = draw.textbbox((0, 0), icon, font=font_obj)
            boxw = box[2] - box[0]
            boxh = box[3] - box[1]
        else:
            boxw, boxh = draw.textsize(icon, font=font_obj)
```

Didn't test it but it should solve the issue. It should be forward ported until saas-16.4